content/speakers/katja-mordaunt.md

```diff
@@ -0,0 +1,7 @@
+{
+    "title": "Katja Mordaunt"
+}
+---
+We all know writing code isn't the hard part. The most important decision we make is what to build - not how to build it. Once we start building, we should continuously reflect on our users, and concentrate on laying good foundations for our future selves. We can better do that as a team of equals.
+
+It's great to have Elm in the eco-system making it less risky than ever for all members of the team to contribute lines of code. I'm on a mission to break down the barrier between those who git and those who do not. We can achieve (or aim for) the ultimate [insert meaningful word for 'agile' here] collaboration. I'll share my story about how my team iterated on a small Elm project over a couple of years and how we experienced continuous small benefits from allowing each other to contribute in our own ways.
```

Comment:
Am being told by my proofreading friend that eco-system is one word. Can we please change to ecosystem?

Reply:
yup, done!